client/client_test.go

```diff
@@ -5437,6 +5438,70 @@ func testBasicGhaCacheImportExport(t *testing.T, sb integration.Sandbox) {
 	testBasicCacheImportExport(t, sb, []CacheOptionsEntry{im}, []CacheOptionsEntry{ex})
 }
 
+func testRegistryEmptyCacheExport(t *testing.T, sb integration.Sandbox) {
```

Comment:
6 lines of code, 65 lines of tests. That seems about right.

That said, I'm not sure if we need a similar test for the local exporter, since the change was in shared code.